x/dsmr/block.go

```diff
@@ -124,8 +125,10 @@ func (b Block) GetID() ids.ID {
 }
 
 // ExecutedBlock contains block data with any referenced chunks reconstructed
-type ExecutedBlock[T Tx] struct {
+type ExecutedBlock[T Tx[T]] struct {
 	BlockHeader
 	ID     ids.ID
-	Chunks []Chunk[T] `serialize:"true"`
+	Chunks []Chunk[T] `canoto:"repeated value,1"`
```

Comment:
Similarly to `Block` - is it correct for us to only serialize the `Chunks` here?

Reply:
We probably don't have to technically serialize this... we know the `ID` from the `Block` this is derived from